inlanecruising_plugin/src/inlanecruising_plugin_node.cpp

```diff
@@ -105,9 +105,7 @@ namespace inlanecruising_plugin
       {"lat_accel_multiplier", config_.lat_accel_multiplier},
       {"back_distance", config_.back_distance},
       {"buffer_ending_downtrack", config_.buffer_ending_downtrack},
-      {"/vehicle_acceleration_limit", config_.max_accel},
-      {"/vehicle_lateral_accel_limit", config_.lateral_accel_limit},
-      {"buffer_ending_downtrack", config_.buffer_ending_downtrack}}, parameters);
+      {"buffer_ending_downtrack", config_.buffer_ending_downtrack}}, parameters); // Global acceleration limits not allowed to dynamically update
```

Comment:
`{"buffer_ending_downtrack", config_.buffer_ending_downtrack}` is duplicated, I don't think we need it

Reply:
good catch. fixed